book/website/community-handbook/pathways/pathways-example.md

```diff
@@ -7,7 +7,7 @@ These examples emphasise the importance of community engagement and the strategi
 (ch-pathways-example-dswb)=
 ## Example 1: Data Science Without Borders (DSWB) - Community-Driven Curation
 
-The [DSWB pathway](https://book.the-turing-way.org/pathways/data-science-without-borders), developed for an international network in Africa, prioritised inclusivity and direct relevance to diverse project requirements.
+The [DSWB pathway](https://book.the-turing-way.org/pathways/pathways-data-science-without-borders), developed for an international network in Africa, prioritised inclusivity and direct relevance to diverse project requirements.
```

Comment:
We should use a cross reference here instead of a URL.

Reply:
We can also use the MyST sytnax, which is a bit cleaner and more consistent with other types of link/reference,

```Markdown
[DSWB pathway](#pw-data-science-without-borders)
```